tex/chapters/MachineLearning.tex

```diff
@@ -1,3 +1,267 @@
 %!TEX root = ../main.tex
 
 \chapter{Machine Learning}\label{cha:machinelearning}
+
+In this chapter a new approach to identifying strategies that are the same will be presented.
+A new type of tournament will be presented, and the summary of the results of this tournament will then be used to train a Machine Learning model.
```

Comment:
It's not a new type of tournament, it's the exact same tournament as for Ashlock fingerprinting. The whole methodology is based on the fact that Ashlock summarised the tournament with an analytical expression (or equivalently an image), whereas you're using a statistical analysis.